doc/src/modules/solvers/solveset.rst

```diff
@@ -474,11 +496,12 @@ How do we deal with cases where only some of the solutions are known?
 What is the plan for solve and solveset?
 ----------------------------------------
 
-There are still a few things ``solveset`` can't do, which ``solve`` can, such
-as solving nonlinear multivariate & LambertW type equations.  Hence, it's not
-yet a perfect replacement for ``solve``. As the algorithms in ``solveset``
-mature, ``solveset`` may be able to be used within ``solve`` to replace some of
-its algorithms.
+Since the goals for ``solveset`` are different from ``solve``,
+``solveset`` is not intended to be a perfect replacement
+for ``solve``. Rather, ``solveset`` should be viewed as complimentary.
+
+As such, there are no current plans to remove ``solve``.
```

Comment:
I think we should just remove this section.

Reply:
That is truly ironic - this is the very section that contained the word 'yet' that started all of this.

But you are right. Once the other items we have been working were addressed, the existence of this section does not make sense.

Consider it on the cutting floor.